### PR TITLE
[MIBM-219][JR] /goods-over-threshold

### PR DESCRIPTION
**Pull request by jordanrowe**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -253,6 +253,28 @@ goodsRemoved.heading = You removed your goods
...
+goodsOverThreshold.p2.a.href = https://www.gov.uk/guidance/making-a-full-import-declaration
+goodsOverThreshold.p3 = If you want to use this service, you need to either:
+goodsOverThreshold.p4 = Remove goods from this declaration
+goodsOverThreshold.p5 = Make a new declaration
+goodsOverThreshold.h3 = How we calculated the value of your goods
+goodsOverThreshold.p6 = The total value of your goods is {0}.
+goodsOverThreshold.p7 = We converted this amount at a rate of:
+goodsOverThreshold.li = {0} to 1 (GBP).
+goodsOverThreshold.p8.1 = We use
+goodsOverThreshold.p8.a.text = HMRC’s monthly exchange rates (opens in new tab)
```
Comment (PaulHodgson) [inline conf/messages:274]: does the apostrophe need to be escaped?

File: test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala [hunk 2]
```diff
@@ -74,6 +74,16 @@ trait CoreTestData {
   val importJourneyWithTwoCompleteGoodsEntries: DeclarationJourney =
     startedImportToGreatBritainJourney.copy(goodsEntries = completedDeclarationJourney.goodsEntries)
 
+  val importJourneyWithGoodsOverThreshold: DeclarationJourney =
+    startedImportToGreatBritainJourney.copy(
+      goodsEntries = GoodsEntries(completedGoodsEntry.copy(
+        maybePurchaseDetails = Some(PurchaseDetails("10000", Currency("Eurozone", "Euro", "EUR"))))))
```
Comment (PaulHodgson) [inline test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala:80]: Probably better to set the to be just over the threshold.

File: test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala [hunk 3]
```diff
@@ -74,6 +74,16 @@ trait CoreTestData {
...
+
+  val completedImportJourneyWithGoodsOverThreshold: DeclarationJourney =
```
Comment (PaulHodgson) [inline test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala:82]: You could probably remove the duplication too.

File: test/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/DeclarationSpec.scala [hunk 4]
```diff
@@ -177,6 +177,7 @@ class DeclarationSpec extends BaseSpec with CoreTestData {
           Some(Declaration(
             sessionId,
             DeclarationType.Import,
+            GoodsDestinations.NorthernIreland,
```
Comment (PaulHodgson) [inline test/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/DeclarationSpec.scala:180]: cool!

File: app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/Declaration.scala [hunk 5]
```diff
@@ -185,14 +185,15 @@ case class DeclarationJourney(sessionId: SessionId,
         maybeCustomsAgent.isDefined || maybeIsACustomsAgent.contains(No)
 
     for {
+      goodsDestination <- maybeGoodsDestination
```
Comment (PaulHodgson) [inline app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/Declaration.scala:188]: cool

File: conf/messages (hunk 1 above)
Comment (jordanrowe) [reply conf/messages:274]: no it's a ’ rather than a '

Comment (PaulHodgson) [reply conf/messages:274]: Nope - looks good

File: test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala (hunk 3 above)
Comment (jordanrowe) [reply test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala:83]: pushed

File: test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala (hunk 2 above)
Comment (jordanrowe) [reply test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala]: have changed to 1915 which is just over £1500 * our stubbed conversion rate of 1.2763

Comment (PaulHodgson) [reply test/uk/gov/hmrc/merchandiseinbaggagefrontend/CoreTestData.scala]: Cool - might be even better to do this as a calculation in the spec to show the relationship, e.g.

val amountOverTheThreshold = 1501 * CurrencyConversionStub.euroConversionRate

and use that as the amount if you see what I mean.

but approved anyways...

**Review by PaulHodgson — APPROVED** [review]

**Pull request merged**